tasks/ca/hierarchy.py

```diff
@@ -0,0 +1,136 @@
+from tasks.hierarchy import (DenormalizedHierarchy, Hierarchy, HierarchyChildParentsUnion, HierarchyInfoUnion,
+                             HierarchyChildParent, GetParentsFunction, LevelHierarchy, LevelInfo)
+from tasks.ca.statcan.geo import (GEO_PR, GEO_CD, GEO_FSA, GEO_CMA, GEO_CSD, GEO_DA, GEO_CT, GEO_DB,
+                                  Geography)
+from lib.logger import get_logger
+
+LOGGER = get_logger(__name__)
+COUNTRY = 'ca'
+
+
+def _levels():
```

Comment:
Why a function instead of a constant?

Reply:
The original one at US was a function and that design was propagated to other countries. But even US one can be a constant :+1: . Changing.